.travis.yml

```diff
@@ -37,6 +37,11 @@ matrix:
       script:
         - ./travis/onnx_integration_tests.sh
 
+    - name: "Pytorch Artifact"
+      python: "3.6"
+      script:
+        - ./travis/run-pytorch-tests.sh
```

Comment:
could we follow the current naming convention here and name it `pytorch_integration_tests.sh` instead?  sorry that the issue description was a bit out dated